### PR TITLE
Added windows compatibility

### PR DESCRIPTION
**Pull request by khoroshevskyi**

### CODE REVIEW

File: ubiquerg/system.py [hunk 1]
```diff
@@ -25,9 +26,21 @@ def is_command_callable(cmd):
         raise ValueError("Empty command to check for callability")
     if os.path.isdir(cmd) or (os.path.isfile(cmd) and not os.access(cmd, os.X_OK)):
         return False
-    # Use `command` to see if command is callable, and rule on exit code.
-    check = "command -v {0} >/dev/null 2>&1 || {{ exit 1; }}".format(cmd)
-    return not bool(os.system(check))
+    if os.name == "nt":
```
Comment (nsheff) [inline ubiquerg/system.py:30]: can you add a comment here to explain what you're doing?

**Comment by nsheff** [comment]
one small suggestion, then go ahead and merge

File: ubiquerg/system.py (hunk 1 above)
Comment (khoroshevskyi) [reply ubiquerg/system.py:29]: it is just linting

Comment (khoroshevskyi) [inline ubiquerg/system.py:30]:
```suggestion
    # if system is windows run this command:
    if os.name == "nt":
```

**Pull request merged**